src/wrapper/jnienv.rs

```diff
@@ -298,6 +298,23 @@ impl<'a> JNIEnv<'a> {
         })
     }
 
+    /// Creates a new local reference frame, in which at least a given number of local
+    /// references can be created.
+    ///
+    /// Contrary to the `PushLocalFrame` from the `JNI` this function returns `Err` instead
+    /// of a negative value on failure.
+    pub fn push_local_frame(&self, capacity: i32) -> Result<()> {
```

Comment:
> from the `JNI`

Do you mean the standard JNI C interface?

Reply:
Yes, should I clarify it somehow?

Reply:
I checked the original JNI documentation, it says
> Returns 0 on success, a negative number **and** a pending OutOfMemoryError on failure.

I think that jni-rs users are interested in what behaviour to expect from this library rather than the difference between it and C-JNI (the original one).

Maybe write something like: «Returns `Err` on failure, with a pending `OutOfMemoryError`». Native devs _have_ to handle the former and _might_ handle the latter (though it might not be a good idea to hide OoME).